bond_app/datastore_cache_api.py

```diff
@@ -47,7 +57,11 @@ def delete_expired_entries():
 
     @staticmethod
     def _build_cache_key(key, namespace):
-        """Create an ndb Key for the key and namespace."""
+        """
+        Create an ndb Key for the key and namespace.
+        Raises InvalidArgument if the cache key is invalid. Note Datastore string cache keys
+        must be at most 1500 bytes.
```

Comment:
Unfortunately JWTs can exceed 1500 bytes which means they will not be cacheable in Datastore.